test/e2e/scaffold/httpbin.go

```diff
@@ -112,3 +115,32 @@ func (s *Scaffold) ScaleHTTPBIN(desired int) error {
 	}
 	return nil
 }
+
+// WaitAllHTTPBINPods waits until all httpbin pods ready.
+func (s *Scaffold) WaitAllHTTPBINPoddsAvailable() error {
```

Comment:
why not use `k8s.WaitUntilNumPodsCreatedE(...)`?

Reply:
It just checks number of Pods are created, not number of Pods are ready.

What we need is the latter, since only a Pod is ready, can it exists in the Endpoints ready addresses.